orchestrator/examples/common.sh

```
@@ -0,0 +1,48 @@
+# Create a new file: orchestrator/examples/common_functions.sh
+#!/bin/bash
```

Shellcheck (0.10.0)

[error] The shebang must be on the first line. Delete blanks and move comments.

(SC1128)